spec/02-integration/03-dao/04-constraints_spec.lua

```diff
@@ -3,11 +3,11 @@ local Factory = require "kong.dao.factory"
 local utils = require "kong.tools.utils"
 
 local api_tbl = {
-  name = "mockbin",
-  hosts = { "mockbin.com" },
-  uris = { "/mockbin" },
-  strip_uri = true,
-  upstream_url = "https://mockbin.com"
+  name         = "example",
```

Comment:
Thanks for the review. I have addressed your comments. I left this PR as two commits, one with the code changes and another one for the changes in style.